src/SortPeopleByDateOfBirth.java

```diff
@@ -0,0 +1,25 @@
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortPeopleByDateOfBirth {
+
+  static void sortPeopleByDateOfBirth(){
```

Comment:
Instead of redeclaring and creating a new list of people below, can you think about how you might **_supply_** this from the list that is creating in the Main class?